charmcraft.yaml

```diff
@@ -0,0 +1,18 @@
+type: charm
+bases:
+    - build-on:
+        - name: ubuntu
+          channel: "20.04"
+          architectures: ["amd64"]
+      run-on:
+        - name: ubuntu
+          channel: "20.04"
+          architectures:
+              - amd64
+        - name: ubuntu
+          channel: "18.04"
+          architectures:
+              - amd64
+parts:
+  charm:
+    charm-python-packages: [setuptools < 58]
```

Comment:
why is this necessary? I don't see jinja2 or charmhelpers in requirements.txt

Reply:
good catch, thanks! i've removed this, and verified that it doesn't cause an issue when I build locally.